### PR TITLE
refactor(sheet): copy&paste

### PR DESCRIPTION
**Pull request by waterhydroxyl**
1、Remove the use of getComputedStyle API and mount a large amount of text strings to the DOM.
2、Fix the memory leak issue caused by pasting external content
3、Keep the style data of each cell in the merged cell consistent when pasting, and improve the consistency of the direct copy and paste scene with Excel and Google Sheets

### before change
b9119be1142f80ee40bf122e3bea41e

### after change
aa1994e6cc59b14a99ce6f0db163dab

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Comment by waterhydroxyl** [comment]
@ybzky

File: packages/core/src/sheets/worksheet.ts [hunk 1]
```diff
@@ -296,9 +296,9 @@ export class Worksheet {
                     });
                 }
 
-                if (row !== startRow || col !== startColumn) {
-                    returnCellMatrix.realDeleteValue(row, col);
-                }
+                // if (row !== startRow || col !== startColumn) {
```
[the remark this answers is not in the document]
Comment (waterhydroxyl) [reply packages/core/src/sheets/worksheet.ts:299]: This change is to fix the issue with internal copy-paste formatting.
image

[the remark this answers is not in the document]
Comment (waterhydroxyl) [reply packages/core/src/sheets/worksheet.ts:299]: For the data model, I think that each merged cell should retain a consistent style; however, rich text or formula information only needs to be stored in the main cell.

[the remark this answers is not in the document]
Comment (ybzky) [reply packages/core/src/sheets/worksheet.ts:299]: Sorry for the late reply. the changes here are not as expected, this API is used in some places at the same time，our convention is that the value of the merged cell is only saved in the upper left cell

[the remark this answers is not in the document]
Comment (waterhydroxyl) [reply packages/core/src/sheets/worksheet.ts]: ok, I have reverted the changes to this part and resolved the issue in the `_pasteInternal `function.

**Review by ybzky — APPROVED** [review]

**Pull request merged**